### PR TITLE
[KernelGen][Nvidia] Add bf16_paged_mqa_logits fused operator with Triton kernel

### PR DESCRIPTION
**Pull request by Yangyang0906C**
## Summary
Adds a Triton kernel for \`bf16_paged_mqa_logits\` fused operator. Computes multi-head weighted ReLU attention logits on paged BF16 KV cache, used in decode-phase inference (e.g., SGLang/vLLM).

Key features:
- Shape-specialized kernels for H=32 and H=64 (D=128, block_size=64) with zero-constexpr dispatch
- Hardcoded shape constants as source-level literals for fastest Triton JIT dispatch
- Supports variable context lengths and paged KV cache layout

## Testing
- Parametrized tests over batch_size, next_n, num_heads (32/64), context_len (64–4096)
- Variable context length test with mixed batch
- Validated against pure PyTorch reference implementation
- 13/13 tests PASSED

## Performance
Test command: \`pytest benchmark/test_bf16_paged_mqa_logits.py --level core\` (NVIDIA H20)
Baseline: DeepGEMM CUDA kernel

**bfloat16:**

| Shape (B, next_n, H, D, ctx_len) | Baseline (ms) | Gems (ms) | Speedup |
|---|---|---|---|
| (1, 1, 32, 128, 1024) | 0.00867 | 0.00787 | 1.102x |
| (4, 1, 32, 128, 1024) | 0.00906 | 0.00778 | 1.165x |
| (8, 1, 32, 128, 2048) | 0.01014 | 0.00960 | 1.057x |
| (16, 1, 32, 128, 4096) | 0.01488 | 0.01517 | 0.981x |
| (4, 2, 32, 128, 2048) | 0.01478 | 0.00906 | 1.633x |
| (1, 1, 32, 128, 4096) | 0.00838 | 0.00806 | 1.040x |
| (4, 1, 64, 128, 2048) | 0.01507 | 0.00870 | 1.732x |
| **Arithmetic Mean** | — | — | **1.244x** |

## Files Changed
- \`src/flag_gems/fused/bf16_paged_mqa_logits.py\`: Triton kernel implementation with H=32 and H=64 specializations
- \`tests/test_bf16_paged_mqa_logits.py\`: Accuracy tests with PyTorch reference
- \`benchmark/test_bf16_paged_mqa_logits.py\`: Performance benchmark
- \`src/flag_gems/fused/__init__.py\`: Register import and \`__all__\`

### CODE REVIEW

File: src/flag_gems/fused/bf16_paged_mqa_logits.py [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+        mask = pos_range < (ctx_len - kv_pos)
+        tl.store(
+            logits_ptr + out_base + pos_range,
+            logits_val,
+            mask=mask,
+            eviction_policy="evict_first",
+        )
+
+
+def bf16_paged_mqa_logits(
```
Comment (tengqm) [inline src/flag_gems/fused/bf16_paged_mqa_logits.py:154]: Please register the `bf16_paged_mqa_logits` operator to the `conf/operators.yaml` file.

**Review by huangyiqun — APPROVED** [review]
